src/Bicep.Core.Samples/Files/LoadFunctions_CRLF/main.bicep

```diff
@@ -89,3 +89,32 @@ var testJsonNestedString2_1 = testJsonObject2_1.nestedString
 var testJsonNestedString2_2 = loadJsonContent('./Assets/test.json.txt', '.object.nestedString')
 
 var testJsonTokensAsArray = loadJsonContent('./Assets/test2.json.txt', '.products[?(@.price > 3)].name')
+
+var testYaml = loadYamlContent('./Assets/test.yaml.txt')

```

Comment:
Baselines are great for visually analyzing the impact of a change, but I wouldn't 100% count on them to verify the end-to-end behavior as they tend to generate large changesets which can often be overlooked.

Could you take a look over the tests in https://github.com/Azure/bicep/blob/069e8b7a46b0a9926dd86cd421ee62e799c12a85/src/Bicep.Core.IntegrationTests/Scenarios/LoadFunctionsTests.cs#L25 and add some of your own for `loadYamlContent()`?

Off the top of my head, some things that would be worth having detailed tests for:
* Any error paths where diagnostics are raised
* End-to-end compilation
* Any edge cases you can think of:
    * empty yaml file
    * single string yaml file
    * unusual line endings
    * different forms of scalars (plain, quoted, numbers with spaces, floats)
    * multi-document yaml files
    * unusual features
* JSON path access

Reply:
Added tests to LoadYamlContent based on LoadJsonContent.